wiki/en/Running-a-Server.md

```diff
@@ -203,34 +182,33 @@ When the Server is running, the operating system will show an icon in the system
+If you are running a Server on your home network, people from outside will not be able to see things inside it. So if you want to operate a Jamulus Server, you need to open a hole in your router to let Jamulus clients connect to it.
 
 **The default port for the current version of Jamulus is UDP (not TCP) port 22124.**
 
 The exact setup of port forwarding differs for every router. For help see [portforward.com](https://portforward.com).
 
-Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your server (_but see also the note on dynamic DNS below_). You yourself have to connect to your own computer instead, since your computer runs the server. Therefore **only you** need to connect to `127.0.0.1`.
+Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself have to connect to your own computer instead, since your computer runs the Server. Therefore **only you** need to connect to `127.0.0.1`.
 
 ## Dynamic DNS and why you will probably need it
 
-Most domestic Internet connections will change their IP address after a period (hours, days or weeks). To make it easier for people to connect to you, you may therefore also want set up a dynamic DNS address. You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
+Most domestic Internet connections will change their IP address after a period (hours, days or weeks). To make it easier for people to connect to you, you may therefore also want set up a dynamic DNS address. You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
 
-Note also that your home router may also change the IP address of the machine that you are running your server on. In which case you may need either to give that machine a static IP in the router's DHCP configuration, or simply port forward to all the addresses in your LAN.
+Note also that your home router may also change the IP address of the machine that you are running your Server on. In which case you may need either to give that machine a static IP in the router's DHCP configuration, or simply port forward to all the addresses in your LAN.
```

Comment:
> or simply port forward to all the addresses in your LAN.

Incoming packet to 221.123.1.23 port 22124.
Hits the router external, goes into the port forwarding table.

This suggests it then goes out to 192.168.1.1 through to 192.168.1.254, subject to removing itself from that list.

Does such a capability _really_ exist?  Every incoming packet on 22124 sent to _every_ machine on the LAN?!  What if they're all running Jamulus on port 22124?

Reply:
Some home routers seem to let you put in a forwarding port and no IP? Dunno.

Reply:
Yes, but I think what this is saying is not a feature normal consumer routers have. Please remove/rephrase this